test/docs/promises.test.js

```diff
@@ -24,11 +24,14 @@ describe('promises docs', function () {
   });
 
   /**
-   * Mongoose async operations, like `.save()` and queries, return
-   * [ES6 promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise).
-   * This means that you can do things like `MyModel.findOne({}).then()` and
+   * Mongoose async operations, like `.save()` and queries, return either an
```

Comment:
Instead of listing out all possible return values, lets instead make it say we return a [thenable](https://promisesaplus.com/#terminology). There are other async operations that return non-promises, like `mongoose.createConnection()`.